docs/faq.md

```diff
@@ -91,8 +91,11 @@ const vitePluginCoolCss = {
 
 For reference, check out [windicss](https://github.com/windicss/vite-plugin-windicss/blob/517eca0cebc879d931c6578a08accadfb112157c/packages/vite-plugin-windicss/src/index.ts#L167).
 
-### What is `The requested module 'xxx' does not provide an export named 'yyy'`?
+### What is going on with vite and `Pre-bundling dependencies:`?
```

Comment:
The docs currently capitalize special names, like Vite, CJS, ES, API. I think we should do that here too. (and some other spots below)